src/probnum/random_variables/_scipy_stats.py

```diff
@@ -0,0 +1,225 @@
+from typing import Any, Dict, Union
+
+import numpy as np
+import scipy.stats
+
+from probnum import utils as _utils
+from . import _random_variable
+from . import _normal
+
+
+_ValueType = Union[np.generic, np.ndarray]
+
+
+class _SciPyRandomVariableMixin:
+    @property
+    def scipy_rv(self):
+        return self._scipy_rv
+
+
+class WrappedSciPyRandomVariable(
```

Comment:
Currently this class is relevant if one uses an RV that has an implementation in SciPy but not in ProbNum. Long term these should converge to zero. Should this class then be removed? Do I understand correctly that the only advantage of having these wrappers over just returning a `RandomVariable` is the access to the Scipy object?

Reply:
Yeah, so one advatage is, as you say, that you can access the scipy object.
I'd also say that it is convenient to have specific types for wrapped scipy random variables because you can now filter them out by their types in interfaces, etc. This might come in handy at a later point.

I would say that we should keep these classes even if we implement all scipy random variables because afterall scipy might decide to add more RVs in future versions. Then we can still support these rvs even if we don't yet have a `probnum` counterpart.

Reply:
I would say that we should keep these classes even if we implement all scipy random variables because afterall scipy might decide to add more RVs in future versions. Then we can still support these rvs even if we don't yet have a `probnum` counterpart.